devops/kubernetes/misc/origin-scripts.yaml

```diff
@@ -0,0 +1,9 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: origin-scripts
+spec:
+  containers:
+    - name: origin-scripts
+      image: "gcr.io/origin-214503/dev/origin-cron:latest"
+      command: ['sleep', '3600']
```

Comment:
The container will exit when this command completes, so `sleep 3600` will only keep it alive for 10 minutes. I think we probably want longer here to handle rewards payouts etc, might need replacing with an infinite loop.

Reply:
Does it matter if someone's connected to it?  Guess it could be an infinite bash loop?

    while true; do foo; sleep 3600 done

Reply:
It doesn't care about connections, it'll just boot them off if it has finished doing its business. Yep that should work fine.